networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,16 +13,58 @@ These release notes track the development of the Network Observability Operator
+
+[id="multi-arch-1.3"]
+==== Multiple architectures now supported 
+* Network Observability Operator can now run on an amd64, ppc64le, or arm64 architecture. Previously, it only ran on x86_64. 
+
+[id="network-observability-operator-1.3.0-bug-fixes"]
+=== Bug fixes
+* Previously, when the Operator was installed from the CLI, the `Role` and `RoleBinding` that are necessary for the Cluster Monitoring Operator to read the metrics were not installed as expected. The issue did not occur when the operator was installed from the web console. Now, either way of installing the Operator installs the required `Role and `RoleBinding. (link:https://issues.redhat.com/browse/NETOBSERV-1003[*NETOBSERV-1003*])
+
+* Since version 1.2, the Network Observability Operator can raise alerts when a problem occurs with the flows collection. Previously, due to a bug, the related configuration to disable alerts, `spec.processor.metrics.disableAlerts` was not working as expected and sometimes ineffectual. Now, this configuration is fixed so that it is possible to disable the alerts. (link:https://issues.redhat.com/browse/NETOBSERV-976[*NETOBSERV-976*])
+
+* Previously, when Network Observability was configured with `spec.loki.authToken` set to `DISABLED`, only a `kubeadmin` cluster administrator was able to view network flows. Other types of cluster administrators received authorization failure. Now, any cluster administrator is able to view network flows. (link:https://issues.redhat.com/browse/NETOBSERV-972[*NETOBSERV-972*])
+
+* Previously, a bug prevented users from setting `spec.consolePlugin.portNaming.enable` to `false`. Now, this setting can be set to `false` to disable port-to-service name translation. (link:https://issues.redhat.com/browse/NETOBSERV-971[*NETOBSERV-971*])
+
+* Previously, the metrics exposed by the console plugin were not collected by the Cluster Monitoring Operator (Prometheus), due to an incorrect configuration. Now the configuration has been fixed so that the console plugin metrics are correctly collected and accessible from the {product-title} web console. (link:https://issues.redhat.com/browse/NETOBSERV-765[*NETOBSERV-765*])
+
+//The below known issue is a WIP. Still trying to understand possible workarounds.
+[id="network-observability-operator-1.3.0-known-issues"]
+=== Known issues
+* When `processor.metrics.tls` is set to `PROVIDED` in the `FlowCollector`, the `flowlogs-pipeline` `servicemonitor` is not adapted to the TLS scheme. (link:https://issues.redhat.com/browse/NETOBSERV-1070[NETOBSERV-1070])
+
+* There was a 403 response from Loki when `authToken` was set to `HOST` and Loki 5.7.2 or above was in use. As a consequence, the `HOST` setting has been deprecated. Only the `FORWARD` setting should be used. (link:https://issues.redhat.com/browse/NETOBSERV-1084[NETOBSERV-1084])
```

Comment:
@jotak I agree with this...I can add another section header for Deprecated and removed features. Should relate the deprecation to the known issue at all? Or just have the deprecation mentioned in the new heading and then separately say, there is a known issue with the `HOST` setting?

Reply:
I think it's fine to relate the two of them, e.g. saying "HOST is deprecated blah blah AND it has a known issue etc."
Actually, whatever feels more natural to you, I think :)

Reply:
how about notifying existing users who already may have HOST configured, may be this can serve as a notice to them until it's fully removed as part of [NETOBSERV-1084](https://issues.redhat.com/browse/NETOBSERV-1084) bug?

Reply:
That's a good suggestion @memodi. We also mention it in the Deprecated features area of the release notes that I just added.

Reply:
Sounds good. Thanks!